pkg/kubectl/util/logs/logs.go

```diff
@@ -27,19 +28,20 @@ import (
 
 var logFlushFreq = pflag.Duration("log-flush-frequency", 5*time.Second, "Maximum number of seconds between log flushes")
 
-// GlogWriter serves as a bridge between the standard log package and the glog package.
-type GlogWriter struct{}
+// ErrorLogWriter serves as a bridge between the standard log package and the glog package.
```

Comment:
Would it be better or worse to set --log-file=/dev/stderr instead of this?  It seems less invasive and maybe we could drop the SetOutput stuff ?

Reply:
In doing it this way, do you lose teh formatted prefix (timestamp, etc) on each line?  Or if not, and you send to another logger, do you log two timestamps?